contrib/ansible/inventory/hosts.multiple.package_from_self_repo

```diff
@@ -0,0 +1,19 @@
+[skydive:children]
+analyzers
+agents
+
+[skydive:vars]
+skydive_listen_ip=0.0.0.0
+# For install package from self repo.
+#skydive_deployment_mode="package_from_self_repo"
+#skydive_repo_baseurl=""
```

Comment:
I would just use `skydive_repo_baseurl` + `skydive_deployment_mode` = `package`, no real need to add specify deployment mode ?

Reply:
This is simple way.
I think if user skydive_repo_baseurl + skydive_deployment_mode = package than need refactored task package.yml